accounting/payables/misc.rst

```diff
@@ -5,5 +5,4 @@ Miscellaneous
 .. toctree::
    :titlesonly:
 
-   misc/forecast
-   misc/employee_expense
```

Comment:
According to your commit message, a redirect rule is needed here.

Reply:
I wondered if to do it or not, as these were two separate files. 
I saw it as a removal, not as a merge, but you're right :) Thanks